index.html

```diff
@@ -80,104 +80,10 @@ <h2>Multi-Post Stories</h2>
                     <a><div class="tag-btn">Ruby</div></a>
                 </div>
                 <div class="works__action-btn">
-                    <button type="button" class="works-btn">See Project</button>
+                    <button type="button" class="works-btn project-btn">See Project</button>
                 </div>
```

Comment:
Kindly add a popup for the "Multi-Post Stories" project as well. Currently, nothing happens when I click on the "See Project" button.

![image](https://user-images.githubusercontent.com/38088818/173885922-cc1650a4-9050-4e02-bde0-727b3eddcb09.png)